test/test_t8code_2d.jl

```diff
@@ -30,7 +30,19 @@ mkdir(outdir)
     end
 end
 
+@trixi_testset "test check_for_negative_volumes" begin
+    # Unstructured mesh with six cells which have left-handed node ordering.
+    mesh_file = joinpath(EXAMPLES_DIR, "rectangle_with_negative_volumes.msh")
+    isfile(mesh_file) ||
+        download("https://gist.githubusercontent.com/jmark/bfe0d45f8e369298d6cc637733819013/raw/cecf86edecc736e8b3e06e354c494b2052d41f7a/rectangle_with_negative_volumes.msh",
+                 mesh_file)
+
+    # This call should throw a warning about negative volumes detected.
+    mesh = T8codeMesh(mesh_file, 2)
```

Comment:
Can we test the warning, e.g., using `@test_logs`?

Reply:
I opted for `@test_warn` instead.

Reply:
[According to the docs](https://docs.julialang.org/en/v1/stdlib/Test/#Test.@test_warn) you should use `@test_logs` if you stick with a warning, cf. the other convo.